Source/EasyNetQ.IntegrationTests/PubSub/When_publish_and_subscribe_with_default_options.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using EasyNetQ.IntegrationTests.Utils;
+using FluentAssertions;
+using Xunit;
+
+namespace EasyNetQ.IntegrationTests.PubSub
+{
+    [Collection("RabbitMQ")]
+    public class When_publish_and_subscribe_with_default_options : IDisposable
+    {
+        public When_publish_and_subscribe_with_default_options(RabbitMQFixture fixture)
+        {
+            bus = RabbitHutch.CreateBus($"host={fixture.Host};prefetchCount=1");
+        }
+
+        public void Dispose()
+        {
+            bus.Dispose();
+        }
+
+        private const int MessagesCount = 10;
+
+        private readonly IBus bus;
+
+        [Fact]
+        public async Task Should_be_able_to_publish_subscribe()
+        {
+            using var timeoutCts = new CancellationTokenSource(TimeSpan.FromSeconds(5));
+
+            var subscriptionId = Guid.NewGuid().ToString();
+            var messagesSink = new MessagesSink(MessagesCount);
+            var messages = MessagesFactories.Create(MessagesCount);
+
+            using (bus.Subscribe<Message>(subscriptionId, messagesSink.Receive))
+            {
+                await bus.PublishBatchAsync(messages, timeoutCts.Token).ConfigureAwait(false);
```

Comment:
I would recommend removing all `ConfigureAwait` from the tests, because they make no sense - there is no syncronization context. Therefore, they only litter the code.